src/gamebryo/gamebryosavegame.h

```diff
@@ -161,10 +167,16 @@ class GamebryoSaveGame : public MOBase::ISaveGame
     float_t readFloat(int bytesToIgnore = 0);
 
     /* Read the plugin list */
-    QStringList readPlugins(int bytesToIgnore = 0);
+    QStringList readPlugins(int bytesToIgnore = 0, bool extraData = false,
```

Comment:
The `corePlugins` parameter should probably be taken by const-ref (applies for the 3 methods).

Reply:
Sure, I can do that tomorrow.

Reply:
I think you forgot the ref-part ;)

Reply:
Ah. Yes. Well I'll need to run it through clang anyway.